packages/react/src/primitives/Alert/Alert.tsx

```diff
@@ -34,10 +34,16 @@ const AlertPrimitive: Primitive<AlertProps, typeof Flex> = (
     }
   }, [setDismissed, onDismiss, dismissed]);
 
+  variation = variation || 'default';
+
   return (
     !dismissed && (
       <Flex
-        className={classNames(ComponentClassNames.Alert, className)}
+        className={classNames(
+          ComponentClassNames.Alert,
+          className,
+          `${ComponentClassNames.Alert}--variation-${variation}`
```

Comment:
We might want to merge your css classes documentation PR before this one and figure out a way to document these classnames. We don't necessarily need to put these classnames in that component classnames object, maybe our solution isn't to add these to that object but dynamically generate them on the docs based on the possible variations and that we follow this specific pattern.